.idea/misc.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@whatzerrname we have to resolve conflicts on .gitignore above maybe to remove this file